common/utils/sendmsg.py

```
@@ -222,16 +217,17 @@ def send_feishu_user(title, content, open_id, user_mail):
             "msg_type": "text",
             "content": {"text": f"{title}\n{content}"},
         }
-        r = requests.post(
+        response = requests.post(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)